docs/core/porting/third-party-deps.md

```diff
@@ -2,22 +2,34 @@
 title: Analyze dependencies to port code to .NET Core
 description: Learn how to analyze external dependencies in order to port your project from .NET Framework to .NET Core.
 author: cartermp
-ms.date: 12/07/2018
+ms.date: 09/23/2019
 ms.custom: seodec18
 ---
 # Analyze your dependencies to port code to .NET Core
 
-To port your code to .NET Core or .NET Standard, you must understand your dependencies. External dependencies are the [NuGet packages](#analyze-referenced-nuget-packages-in-your-projects) or [DLLs](#analyze-dependencies-that-arent-nuget-packages) you reference in your project, but that you don't build. Evaluate each dependency and develop a contingency plan for the ones that aren't compatible with .NET Core. Here's how to determine if a dependency is compatible with .NET Core.
+To port your code to .NET Core or .NET Standard, you must understand your dependencies. External dependencies are the [NuGet packages](#analyze-referenced-nuget-packages-in-your-projects) or [DLLs](#analyze-dependencies-that-arent-nuget-packages) you reference in your project, but that you don't build yourself.
 
-## Analyze referenced NuGet packages in your projects
+## Migrate your packages to `PackageReference`
 
-If you reference NuGet packages in your project, you need to verify if they're compatible with .NET Core.
-There are two ways to accomplish that:
+.NET Core uses [PackageReference](/nuget/consume-packages/package-references-in-project-files) to specify package dependencies. If you're using [packages.config](/nuget/reference/packages-config) to specify your packages in your project, you need to convert it to the `PackageReference` format because `packages.config` isn't supported in .NET Core.
 
-- [Using the NuGet Package Explorer app](#analyze-nuget-packages-using-nuget-package-explorer)
-- [Using the nuget.org site](#analyze-nuget-packages-using-nugetorg)
+To learn how to migrate, see the [Migrate from packages.config to PackageReference](/nuget/reference/migrate-packages-config-to-package-reference) article.
 
-After analyzing the packages, if they're not compatible with .NET Core and only target .NET Framework, you can check if the [.NET Framework compatibility mode](#net-framework-compatibility-mode) can help with your porting process.
+## Upgrade your NuGet packages
+
+After your migrating your project to the `PackageReference` format, you need to verify if your packages are compatible with .NET Core.
+
+First, upgrade your packages to the latest version that you can. This can be done with the NuGet Package Manager UI in Visual Studio. It's likely that newer versions of your package dependencies are already compatible with .NET Core.
+
+## Analyze your package dependencies
+
+If you haven't already verified that your converted and upgraded package dependencies work on .NET Core, there are a few ways that you can achieve that:
+
+* Analyze NuGet packages using nuget.org
```

Comment:
the formatting here is weird and doesn't match the other section. should it be the following or should line 34 be a bulleted list too?

```suggestion
### Analyze NuGet packages using nuget.org
```